[2024-10-15T17:55:58.207+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T17:55:58.247+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.stock_update manual__2024-10-15T17:55:49.882608+00:00 [queued]>
[2024-10-15T17:55:58.268+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.stock_update manual__2024-10-15T17:55:49.882608+00:00 [queued]>
[2024-10-15T17:55:58.269+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-15T17:55:58.306+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): stock_update> on 2024-10-15 17:55:49.882608+00:00
[2024-10-15T17:55:58.321+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=809) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T17:55:58.326+0000] {standard_task_runner.py:72} INFO - Started process 811 to run task
[2024-10-15T17:55:58.327+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'stock_update', 'manual__2024-10-15T17:55:49.882608+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmphy4aj4yy']
[2024-10-15T17:55:58.333+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask stock_update
[2024-10-15T17:55:58.457+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.stock_update manual__2024-10-15T17:55:49.882608+00:00 [running]> on host 2296928c39b5
[2024-10-15T17:55:59.029+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='stock_update' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T17:55:49.882608+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T17:55:49.882608+00:00'
[2024-10-15T17:55:59.031+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T17:55:59.062+0000] {logging_mixin.py:190} INFO - One or more files do not exist. Please check the names and paths.
[2024-10-15T17:55:59.063+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-15T17:55:59.087+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T17:55:59.089+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=stock_update, run_id=manual__2024-10-15T17:55:49.882608+00:00, execution_date=20241015T175549, start_date=20241015T175558, end_date=20241015T175559
[2024-10-15T17:55:59.150+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-15T17:55:59.209+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-15T17:55:59.225+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
